plugin/plugin.py

```diff
@@ -171,6 +172,13 @@ def _handle_status_notification_notification(self, payload: CopilotPayloadStatus
         pass
 
     def request_get_completions(self, view: sublime.View) -> None:
+        self._request_completions(view=view, request=REQ_GET_COMPLETIONS)
+        self.request_get_completions_cycling(view=view)
```

Comment:
I think this means the result of the first request is always unused but it will make the popup flickering?